src/api/events.js

```diff
@@ -37,8 +37,9 @@ export default class Events extends baseRequest {
     return data.data
   }
 
-  add = (event) => {
-    return this.post(this.method, event)
+  add = async (event) => {
+    const response = await this.post(this.methodAll, event)
+    return response
   }
```

Comment:
I think we can do this, but if you wanna keep this function like you have it right now is also fine
```suggestion
  add = async (event) => ( await this.post(this.methodAll, event) )
```

Reply:
Done